unlock_indicator.c

```diff
@@ -213,6 +215,9 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
             case STATE_PAM_WRONG:
                 text = "wrong!";
                 break;
+            case STATE_PAM_FAIL:
+                text = "Failed; exiting";
```

Comment:
Indeed. I'd rather go for a simple "Locking failed", but still is out of bounds. Do you think we could go for a display on two lines ?